azure-pipelines.yml

```diff
@@ -31,6 +31,12 @@ stages:
 - stage: build
   displayName: Build
   jobs:
+  - ${{ if and(ne(variables['System.TeamProject'], 'public'), notin(variables['Build.Reason'], 'PullRequest')) }}:
+    - template: /eng/common/templates/job/onelocbuild.yml
+      parameters:
+        CreatePr: false
+        LclSource: lclFilesfromPackage
+        LclPackageId: 'LCL-JUNO-PROD-SOURCELINK'
```

Comment:
Why is this not part of the common template `jobs.yml`?

Reply:
There are a lot of repos that need it that don't use `jobs.yml` and a lot of repos that use `jobs.yml` that don't need it. It added a bunch more parameters to `jobs.yml` that just got plumbed through to this template so we ultimately decided to just keep it separate.

Reply:
I see only one parameter here that'd need to be passed: the LclPackageId. that also is cryptic to me and I'd think it could be synthesized based on the name of the repo. So really, I think we just need a single parameter: `enableLocalization: true` similar to other `enable*` parameters.

Reply:
Here, that's true. However, across all the repos that use this there's a whole host of different configurations that need to be passed. What's more, the package IDs can unfortunately not be synthesized from the repo name (e.g. deployment-tools where the package ID is `LCL-JUNO-PROD-MAGE`).

We could look at making use of `jobs.yml` in the next iteration (which there is funding for and will start immediately), but simpleloc is being turned off today and pushing out that change will require waiting for an arcade promotion.

Reply:
Following up on it as a separate effort sounds good. I wonder what the other parameters are and if we could avoid them. Also is it possible to change the PackageId to match the repo name - if only to make it easier to decipher what they mean.

Reply:
Thanks!